eahub/templates/eahub/profile.html

```diff
@@ -92,7 +92,17 @@ <h3>Community</h3>
           <div class="list"><div class="tag">Organisational affiliations:</div> {{ profile.get_pretty_organisational_affiliations }}</div>
           {% endif %}
           {% if profile.local_groups.exists %}
-          <div class="list"><div class="tag">Local groups:</div> {{ profile.get_pretty_local_groups }}</div>
+          <div class="list turquoise-links">
+            {% if profile.local_groups.all.count == 1 %}
```

Comment:
There's an easier way. https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#pluralize

Reply:
The point of the if/else is not the pluralising of "Local group/s", but rather to have a block list where no. of groups > 1 and an inline statement where no. of group = 1